### DISCUSSION TITLE
Allow Err to return an error message

### DISCUSSION THREAD

**Discussion by kingsleyh** [mint-lang/mint#335] [Language Development]
It would be helpful if Err could return an error message:

https://github.com/mint-lang/mint-runtime/blob/master/source/Result.js

so that we could do something like this:

```
enum KeyPair.Error {
  KeyPairGenerationError String
} 
return new Err($KeyPair_Error_KeyPairGenerationError, "some error message")
```

not sure if that kind of thing is supported by enum

**Comment by gdotdesign** [comment]
It is possible for a while now :joy: closing this.
